data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaCustomConsumer.java

```diff
@@ -99,6 +99,7 @@ public class KafkaCustomConsumer implements Runnable, ConsumerRebalanceListener
     private final LogRateLimiter errLogRateLimiter;
     private final ByteDecoder byteDecoder;
     private final long maxRetriesOnException;
+    private final Map<Integer, Long> lastReceivedTimeStampMap;
```

Comment:
```suggestion
    private final Map<Integer, Long> partitionToLastReceivedTimestampMillis;
```

Clarifies how we index and that we are storing millis here.